canonical/ubuntu.com
# WD-12825 - rebrand /azure/fips

## Done

Rebrand /azure/fips. No content was changed, layout only

## QA

- [demo link](https://ubuntu-com-14144.demos.haus/azure/fips)
- [figma](https://www.figma.com/design/MinLlXouBnEDqaoAL3bTWC/24.10-Azure-bubble?node-id=1-2233&t=5AOamUxpwm8vWNig-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-12825](https://warthogs.atlassian.net/browse/WD-12825)

[WD-12825]: https://warthogs.atlassian.net/browse/WD-12825?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
